src/Analysis/Ast/Impl/Analyzer/Evaluation/ExpressionEval.Callables.cs

```diff
@@ -383,5 +383,23 @@ private bool EvaluateFunctionBody(IPythonFunctionType fn)
             DeclareVariable(p.Name, paramType.CreateInstance(ArgumentSet.Empty(p.NameExpression, this)),
                 VariableSource.Declaration, p.NameExpression);
         }
+
+        internal void ProcessCallForReferences(CallExpression callExpr) {
+            switch (callExpr.Target) {
+                case NameExpression nex when !string.IsNullOrEmpty(nex.Name):
+                    // Add reference to the function
+                    this.LookupNameInScopes(nex.Name)?.AddReference(GetLocationOfName(nex));
+                    break;
+                case MemberExpression mex when !string.IsNullOrEmpty(mex.Name): {
+                    var t = GetValueFromExpression(mex.Target)?.GetPythonType();
+                    t?.GetMember(mex.Name)?.AddReference(GetLocationOfName(mex));
+                    break;
+                }
+            }
+            // Add references to all arguments.
+            foreach (var arg in callExpr.Args) {
```

Comment:
Should we limit this to user code?

Reply:
We already don't store references in library code, right?

Reply:
In library code no, only user-to-library so user can find, say, all usages of `print`

Reply:
If the current file is a library, then I'd think we can skip it. The only way I can see it breaking is library code which references user code, which I can't think of a way to sanely construct.